source/common/config/grpc_mux_impl.h

```diff
@@ -178,6 +185,7 @@ class GrpcMuxImpl : public GrpcMux,
   XdsResourcesDelegateOptRef xds_resources_delegate_;
   const std::string target_xds_authority_;
   bool first_stream_request_;
+  bool starting_{true};
```

Comment:
nit: starting is too generic. Maybe something in the lines of previously fetched data, or something that will convey the meaning of this

Reply:
Done.